src/components/Explore/utils.ts

```diff
@@ -85,20 +85,20 @@ export function getDateRange(period: Period): Date[] {
 }
 
 export const EXPLORE_METRICS = [
```

Comment:
Might be worth adding a comment like:

// NOTE: We try to keep these alphabetized by metric name to make the trends metric list easier to navigate for users.